src/Features/Core/Portable/GenerateConstructorFromMembers/AbstractGenerateConstructorFromMembersCodeRefactoringProvider.GenerateConstructorWithDialogCodeAction.cs

```diff
@@ -76,10 +76,12 @@ public override object GetOptions(CancellationToken cancellationToken)
                 var addNullChecksOption = result.Options.FirstOrDefault(o => o.Id == AddNullChecksId);
                 if (addNullChecksOption != null)
                 {
+                    // ILegacyGlobalOptionsWorkspaceService is guaranteed to be not null here because we have checked it before the code action is provided.
+                    var globalOptions = _document.Project.Solution.Services.GetRequiredService<ILegacyGlobalOptionsWorkspaceService>();
```

Comment:
❔ Are there any other instances of `GetRequiredService<ILegacyGlobalOptionsWorkspaceService>`?

Reply:
This should be the last one